Source/WebCore/rendering/mathml/RenderMathMLScripts.cpp

```diff
@@ -125,6 +125,10 @@ std::optional<RenderMathMLScripts::ReferenceChildren> RenderMathMLScripts::valid
         //
         // https://www.w3.org/TR/MathML3/chapter3.html#presm.mmultiscripts
         //
+        // The base cannot be an <mprescripts> element.
+        if (isPrescriptDelimiter(*base))
+            return std::nullopt;
+
```

Comment:
Please remember to mention this change in you commit message too.

Reply:
Alright, mentioned